google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
I'm trying to figure out why the `<div>` was added to the element's shadow dom.  Also, `var content = this.children[0];` won't select marker content if it's just text content (`<google-marker>text content</google-marker>`).